.github/workflows/deploy-pr-to-spaces.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           ---
           You can install the changes in this PR by running:
           ```bash
-          pip install https://gradio-builds.s3.amazonaws.com/${{ github.sha }}/gradio-${{ env.GRADIO_VERSION }}-py3-none-any.whl
+          pip install https://gradio-builds.s3.amazonaws.com/${{ github.sha }}/gradio-${{ env.gradio_version }}-py3-none-any.whl
```

Comment:
I'm thinking we can push the build to a separate URL as well that depends on the PR number (so that it's easier to remember) and so that it doesn't change with every commit. Something like (https://gradio-builds.s3.amazonaws.com/pr4893/gradio-${{ env.gradio_version }}-py3-none-any.whl

The reason we use the github sha in the space is because the docker build of spaces caches the install step of requirements.txt so if the requirements.txt doesn't change between commits, the latest commit won't actually be installed